metrics/metrics.go

```diff
@@ -99,7 +99,7 @@ func NewMetrics(subsystem string) *Metrics {
 			Buckets: prometheus.ExponentialBucketsRange(0.05, 1200, 20),
 			Help:    "Histogram of HTTP server request durations",
 		}, []string{
-			"method", "commitment_mode", "DA_cert_version", // no status on histograms because those are very expensive
+			"method", // no status on histograms because those are very expensive
```

Comment:
@hopeyen had to remove this because it was causing a revert in the metric middleware, since you weren't passing these labels (because we don't have access to them before the request when this is called):
```
panic: inconsistent label cardinality: expected 3 label values but got 1 in []string{"GET"}
```

There might be another way to keep these in, but I suggest merging this first and then figuring out that in your other PR which should also change the behavior of the handlers to use custom errors instead of relying on the values returned along with an err.